tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs

```diff
@@ -77,8 +79,17 @@ public void RefreshTrackedRepositories()
 
     public bool AddRepositoryPath(string extensionCLSID, string rootPath)
     {
-        var normalizedPath = rootPath.ToUpper(CultureInfo.InvariantCulture).TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
-        var result = SourceControlIntegration.ValidateSourceControlExtension(extensionCLSID, normalizedPath);
+        var normalizedPath = rootPath.TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
+        var result = SourceControlIntegration.ValidateSourceControlExtension(extensionCLSID, normalizedPath);

```

Comment:
With the recent UI changes merged, these calls to register/unregister will have to be placed differently. The call to register would need to go in _AssignSourceControlProviderToRepository_ and unregister would need to go in
_RemoveTrackedRepositoryFromDevHome_. Thanks!